TEMPLATE/test/test_auth.py

```diff
@@ -0,0 +1,23 @@
+from TEMPLATE.db.fixtures import DEFAULT_PASSWORD
+
+
+def test_login(client_unauthenticated, user, db_session):
+    response = client_unauthenticated.post(
+        "/api/auth/login", json=dict(email=user.email, password=DEFAULT_PASSWORD)
+    )
+    assert response.status_code == 200
+    assert response.json.get("access_token") is not None
```

Comment:
maybe just use
```python
assert response.json.get("access_token")
```
so you can check for both None and empty string?

Reply:
yeah that would be better